source/utilities.tex

```diff
@@ -6463,6 +6464,2042 @@
 \end{example}
 \end{itemdescr}
 
+\rSec1[expected]{Expected objects}
+\indexlibraryglobal{expected}%
+
+\rSec2[expected.general]{In general}
+
+\pnum
+Subclause \ref{expected} describes the class template \tcode{expected}
+that represents expected objects.
+An \tcode{expected<T, E>} object holds
+an object of type \tcode{T} or an object of type \tcode{unexpected<E>} and
+manages the lifetime of the contained objects.
+
+\rSec2[expected.syn]{Header \tcode{<expected>} synopsis}
+
+\indexlibraryglobal{unexpect_t}%
+\indexlibraryglobal{unexpect}%
+\begin{codeblock}
+namespace std {
+  // \ref{expected.un.object}, class template \tcode{unexpected}
+  template<class E> class unexpected;
+
+  // \ref{expected.bad}, class \tcode{bad_expected_access}
```

Comment:
@jensmaurer: should this be "class <ins>template</ins> bad_expected access"?

Reply:
Yes. Are you fixing that?

Reply:
Yes, will do, thanks!